study-builder/studies/study-example.conf

```diff
@@ -82,11 +82,25 @@
   # Each language is a 2-letter ISO language code, e.g. "en", "ru", "fr" etc , language name e.g. "English" and isDefault (true/false)
   # Optional, can be left out or set to null.
   "supportedLanguages": [
+    {
       "language": "string",
       "name": "string",
-      "isDefault": bool
+      "isDefault": "bool"
+    }
   ],
 
+  # Settings that control various aspects of invitations for study, if applicable.
+  #
+  # Optional.
+  "inviteSetting": {
```

Comment:
Other settings I can see us adding here:
* whether to require recaptcha or not
* exact match on invite code or not
* or even whether to check zip code during invite check or not